MessagingService.BusinessLogic/Services/EmailDomainService.cs

```diff
@@ -83,7 +83,38 @@ public class EmailDomainService : IEmailDomainService
             emailAggregate.ReceiveResponseFromProvider(emailResponse.RequestIdentifier, emailResponse.EmailIdentifier);
 
             // Save Changes to persistance
-            await emailAggregateRepository.SaveChanges(emailAggregate, cancellationToken);
+            await this.EmailAggregateRepository.SaveChanges(emailAggregate, cancellationToken);
+        }
+
+        /// <summary>
+        /// Emails the aggregate repository trace generated.
+        /// </summary>
+        /// <param name="trace">The trace.</param>
+        /// <param name="logLevel">The log level.</param>
+        private void EmailAggregateRepository_TraceGenerated(String trace,
+                                                             LogLevel logLevel)
+        {
+            switch(logLevel)
```

Comment:
Codacy found an issue: [Add a 'default' clause to this 'switch' statement.](https://app.codacy.com/manual/stuart_ferguson1/Messaging/pullRequest?prid=5778932)